modules/KalturaSupport/resources/mw.kAnalony.js

```diff
@@ -442,12 +458,28 @@
 				_this.firstPlay = false;
 			}
 			_this.smartSetInterval(function(){
-                if ( !_this._p100Once || (_this.embedPlayer.donePlayingCount > 0)){ // since we report 100% at 99%, we don't want any "VIEW" reports after that (FEC-5269)
-					_this.sendAnalytics(playerEvent.VIEW, {
-                        playTimeSum: _this.playTimeSum,
-                        averageBitrate: _this.rateHandler.getAverage(),
-                        bufferTimeSum: _this.bufferTimeSum
-                    });
+				var droppedFramesRatio;
+				try{
```

Comment:
Move all of the code block to a function and call it from here, no need to manage the different api checks inline.